apps/bondy/src/bondy_utils.erl

```diff
@@ -294,13 +294,19 @@ external_session_id(undefined) ->
     undefined.
 
 %% -----------------------------------------------------------------------------
-%% @doc
+%% @doc It returns the timeout in ms.
+%% - Provided timeout if it is greater than 0
+%% - wamp_max_call_timeout if the provided timeout is equals to 0
+%% - wamp_call_timeout if no timeout is provided
 %% @end
 %% -----------------------------------------------------------------------------
 timeout(#{timeout := T}) when is_integer(T), T > 0 ->
-    T;
```

Comment:
This block is not right. We are introducing ambiguity. Timeouts should be in milliseconds.